FullStackTests/Tests/ManagementFullStackTests.swift

```diff
@@ -60,67 +56,96 @@ class ManagementFullStackTests: XCTestCase {
             let oathSession = try? await OATHSession.session(withConnection: connection)
             if transport == .nfc {
                 XCTAssert(oathSession == nil)
             }
             let managementSession = try await ManagementSession.session(withConnection: connection)
-            guard let enableConfig = deviceInfo.config.deviceConfig(enabling: true, application: .oath, overTransport: .nfc)?.deviceConfig(enabling: true, application: .piv, overTransport: .nfc) else { XCTFail(); return }
+            guard let enableConfig = deviceInfo.config.deviceConfig(enabling: true, application: .OATH, overTransport: .nfc)?.deviceConfig(enabling: true, application: .PIV, overTransport: .nfc) else { XCTFail(); return }
             try await managementSession.updateDeviceConfig(enableConfig, reboot: false)
+            let enabledInfo = try await managementSession.getDeviceInfo()
+            XCTAssert(enabledInfo.config.isApplicationEnabled(.OATH, overTransport: .nfc))
+            XCTAssert(enabledInfo.config.isApplicationEnabled(.PIV, overTransport: .nfc))
             #if os(iOS)
             await connection.nfcConnection?.close(message: "Test successful!")
             #endif
-            let enabledInfo = try await managementSession.getDeviceInfo()
-            XCTAssert(enabledInfo.config.isApplicationEnabled(.oath, overTransport: .nfc))
-            XCTAssert(enabledInfo.config.isApplicationEnabled(.piv, overTransport: .nfc))
         }
     }
     
     func testDisableAndEnableWithHelperOATH() throws {
         runManagementTest { connection, session, transport in
-            try await session.setEnabled(false, application: .oath, overTransport: transport)
+            try await session.setEnabled(false, application: .OATH, overTransport: transport)
             var info = try await session.getDeviceInfo()
-            XCTAssertFalse(info.config.isApplicationEnabled(.oath, overTransport: transport))
+            XCTAssertFalse(info.config.isApplicationEnabled(.OATH, overTransport: transport))
             let oathSession = try? await OATHSession.session(withConnection: connection)
             XCTAssert(oathSession == nil)
             let managementSession = try await ManagementSession.session(withConnection: connection)
-            try await managementSession.setEnabled(true, application: .oath, overTransport: transport)
+            try await managementSession.setEnabled(true, application: .OATH, overTransport: transport)
             info = try await managementSession.getDeviceInfo()
+            XCTAssert(info.config.isApplicationEnabled(.OATH, overTransport: transport))
             #if os(iOS)
             await connection.nfcConnection?.close(message: "Test successful!")
             #endif
-            XCTAssert(info.config.isApplicationEnabled(.oath, overTransport: transport))
         }
     }    
+    
+    func testZNFCRestricted() throws {
```

Comment:
any reason for the Z in the test name?

Reply:
Thanks for explaining!

Reply:
I added a short comment so the next person stumbling upon this strange name will now why.